models/crowdhuman/softmax_entropy_op.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import mxnet as mx
+
+class SoftmaxEntropyOperator(mx.operator.CustomOp):
```

Comment:
Why not use the official version
https://mxnet.apache.org/versions/1.6/api/r/docs/api/mx.symbol.softmax_cross_entropy.html

Reply:
Mxnet does provide a softmax_cross_entropy op, however, in Double Pred, we need to add cls loss and reg loss together before computing EMD loss. Hence, cls loss should not be parsed with `MakeLoss` before adding with reg loss(which was done in official softmax_cross_entropy op). What's more, this op provides with a param `require_grad` to get the gradient of input, but we found that it returns the square of the gradient rather than the original gradient(we don't know the reason why mxnet does this, maybe it's a bug :) ).